napari/utils/misc.py

```diff
@@ -51,17 +51,26 @@ def parse_version(v) -> 'packaging.version._BaseVersion':
 
 
 def running_as_bundled_app(*, check_conda=True) -> bool:
-    """Infer whether we are running as a briefcase bundle."""
+    """Infer whether we are running as a bundle."""
     # https://github.com/beeware/briefcase/issues/412
     # https://github.com/beeware/briefcase/pull/425
     # note that a module may not have a __package__ attribute
     # From 0.4.12 we add a sentinel file next to the bundled sys.executable
+    warnings.warn(
+        trans._(
+            "Briefcase installations are no longer supported as of vX.Y.Z. "
+            "running_as_bundled_app() will be removed in a future release.",
```

Comment:
```suggestion
            "running_as_bundled_app() will be removed in a 0.6.0 release.",
```

Reply:
I'd also need advice on what to put in the `vX.Y.Z` line above :D

Reply:
0.4.18 as it not working on most machines now.